src/promptflow/tests/sdk_cli_test/conftest.py

```diff
@@ -77,19 +79,35 @@ def setup_local_connection(local_client, azure_open_ai_connection):
     global _connection_setup
     if _connection_setup:
         return
-    connection_dict = json.loads(open(CONNECTION_FILE, "r").read())
     if RecordStorage.is_replaying_mode():
```

Comment:
replaying mode should also be able to read json file?

Reply:
There is no json file in replay mode.

Reply:
how did previous test prepare connections then?